stacker/blueprints/info.py

```diff
@@ -0,0 +1,244 @@
+from .variables.types import (
+    CFNType,
+    TroposphereType,
+)
+from ..util import load_object_from_string
+
+
+def format_var_type(var_type):
+    """Format a given var type to be displayed by the info command.
+
+    Args:
+        var_type (type): type of variable
+
+    Returns:
+        str: formatted display variable type
+
+    """
+    is_list = False
+    if isinstance(var_type, TroposphereType):
+        if var_type.many:
+            is_list = True
+
+        var_type = var_type.type
+
+    if isinstance(var_type, (list, tuple)) and len(var_type):
```

Comment:
Not sure I understand why the `len(var_type)` is here?  What's case is this trying to deal with?  Can the var_type be an empty list?